ab_tool/templates/ab_tool/editExperiment.html

```diff
@@ -154,10 +154,11 @@ <h4 class="modal-title">Edit Summary</h4>
       <div class="modal-footer">
         <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
         {% if create %}
-            <button type="button" id="createNow" class="btn btn-primary" ng-click="submit()">Submit Create</button>
+            <button type="button" id="createNow" class="btn btn-primary" ng-if="!submitting" ng-click="submit()">Submit Create</button>
         {% else %}
-            <button type="button" id="createNow" class="btn btn-primary" ng-click="submit()">Submit Update</button>
+            <button type="button" id="createNow" class="btn btn-primary" ng-if="!submitting" ng-click="submit()">Submit Update</button>
         {% endif %}
+        <button enabled="false" type="button" class="btn btn-primary" ng-if="submitting">Please wait...</button>
```

Comment:
Just curious what this button is for... is the idea that you're showing wait text during submission and it will be hidden otherwise?  If so, why does it have to be a button element over, say, a span?